compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -406,6 +568,10 @@ void OMR::CodeGenerator::lowerTrees()
       }
 
    self()->postLowerTrees();
+
+   if (comp()->getOption(TR_SplitWarmAndColdBlocks) &&
+       !comp()->compileRelocatableCode())
+      self()->findLastWarmBlock();
```

Comment:
nitpick (since other changes are needed above anyway): better to add brace brackets since the conditions span multiple lines.